### ISSUE TITLE
Disagreement betweeen the log and configuration of kd-resnet18_from_resnet34

### ISSUE THREAD

**Issue by Calmepro777** [yoshitomo-matsubara/torchdistill#237]
Hi yoshitomo. 

I just noticed that in the config, the initial learning rate is 0.1 while that in the log seemed to be 0.3.

Moreover, the batch size is set to 256 in the config. However, given that there are 1.2 million images in ILSVRC 2012 train set, and from the log there seems to be 1667 batches in total. I guess this means the batch size would be around 720.

**Comment by yoshitomo-matsubara** [comment]
Hi @Calmepro777 

I believe that you're referring to [this log file](https://github.com/yoshitomo-matsubara/torchdistill/blob/main/configs/official/ilsvrc2012/yoshitomo-matsubara/rrpr2020/kd-resnet18_from_resnet34.log).

This experiment was done in distributed training mode with 3 GPUs (See world_size is set to 3 [here](https://github.com/yoshitomo-matsubara/torchdistill/blob/main/configs/official/ilsvrc2012/yoshitomo-matsubara/rrpr2020/kd-resnet18_from_resnet34.log#L2)), thus effective batch size is 256*3=768 and then learning rate is also multiplied by 3, thus 0.1 (in yaml) * 3 = 0.3, as mentioned in my paper and [README](https://github.com/yoshitomo-matsubara/torchdistill/tree/main/configs/official/ilsvrc2012/yoshitomo-matsubara/rrpr2020#configuration)

Also, please use Discussions above (instead of Issues) for questions.
As explained [here](https://github.com/yoshitomo-matsubara/torchdistill#issues--questions--requests), I want to keep Issues mainly for bug reports.
